### ISSUE TITLE
Interest in standardization of image formats, metadata, sensor models for drone imagery across several groups, e.g. Ag DWG, UxS DWG, GeoTIFF SWG, SWE DWG, etc.

### ISSUE THREAD

**Issue by ogcscotts** [ogcscotts/TC-Meeting-topics#23]

**Comment by ayoumans** [comment]
This just came up today in a meeting that a GeoTIFF standard needs to be completed!  We generate a lot of GeoTIFFs in our organization, so this is overdue!  

I must do some research because in the meeting the issue of Axis Order came up in GeoTIFF.  I believe that this was solved in the current draft standards for GeoTIFF.

**Comment by pebau** [comment]
hm, the Coverage Implementation Schema (CIS) provides that: there is a common, format independent model which is harmonized with SWE Common. The encoding as GeoTIFF is well defined. And there is no axis order problem :)

**Comment by rouault** [comment]
> The encoding as GeoTIFF is well defined. And there is no axis order problem :)

As far as I understand it the GeoTIFF coverage encoding profile only addresses axis ordering for the XML part of the coverage, not for the pure GeoTIFF part.  Anyway the correct place to discuss this would be https://github.com/opengeospatial/geotiff/issues/1

**Comment by luciocola** [comment]
Yes but in geotiff you can tag en external file referenced with an UUID and the external file could be like the one we defined in GMLJP2 based on gmlcov... maybe it helps in harmonizing all the encodings
